src/Tizen.NUI/src/public/BaseComponents/ViewEvent.cs

```diff
@@ -1259,5 +1259,25 @@ private void OnWindowWheelEvent(object sender, Window.WheelEventArgs e)
             }
         }
 
+        /// <summary>
+        /// TouchArea can reset the view's touchable area.
```

Comment:
If you want to add new line in the docs, you need `<br/>` tag.
```suggestion
        /// TouchArea can reset the view's touchable area. <br/>
```

| Before | After |
| --- | --- |
|![before](https://user-images.githubusercontent.com/15519251/97825025-dd017d80-1d00-11eb-9b41-7427c9bf9ffc.png) |![after](https://user-images.githubusercontent.com/15519251/97825036-e12d9b00-1d00-11eb-8940-3f6d7c7589e2.png) |